packages/twenty-front/src/modules/object-record/utils/generateEmptyFieldValue.ts

```diff
@@ -100,8 +101,14 @@ export const generateEmptyFieldValue = (
         additionalPhones: null,
       };
     }
+    case FieldMetadataType.RICH_TEXT_V2: {
+      return null;
+    }
+    case FieldMetadataType.TS_VECTOR: {
+      return null;
+    }
     default: {
-      throw new Error('Unhandled FieldMetadataType');
+      return assertUnreachable(fieldMetadataItem.type);
```

Comment:
The `assertUnreachable` function takes a message string as the second parameter. You can use the same as in the old version.